astropy/visualization/mpl_style.py

```diff
@@ -26,12 +24,13 @@
 example), you should reset the matplotlib settings to the library defaults
 *before* applying the astropy style, e.g.::
 
-    >>> import matplotlib as mpl
-    >>> from astropy.visualization import astropy_mpl_style
-    >>> mpl.rcdefaults()
-    >>> mpl.rcParams.update(astropy_mpl_style)
+>>> import matplotlib as mpl
+>>> from astropy.visualization import astropy_mpl_style
+>>> mpl.rcdefaults()
+>>> mpl.rcParams.update(astropy_mpl_style)
 """
 
+__doctest_requires__ = {'*': ['matplotlib']}
```

Comment:
Ah, I see:
https://travis-ci.org/astropy/astropy/jobs/180328163#L2470

Well, there must be a different way to declare `__doctest_requires__` then.
@bsipocz or anyone knowlegeable in such thing - Help!?

It's clear that the module docstring should be place so that it registers as the module docstring with Python, right?